doc/source/theory/pQCD.rst

```diff
@@ -46,23 +46,55 @@ Note the additional minus in the definition of :math:`\gamma`.
+Since scale-dependent contributions to a perturbative prediction are fixed by |RGE| invariance,
+the scale variation can be used to generate higher order contributions, 
+which are then taken as a proxy for the whole missing higher orders.
+This method provides many advantages:
 
-.. math ::
-    \gamma^{(1)}(N) &\to \gamma^{(1)}(N) - \beta_0 \ln(\mu_F^2/\mu_R^2) \gamma^{(0)} \\
-    \gamma^{(2)}(N) &\to \gamma^{(2)}(N) - 2 \beta_0 \ln(\mu_F^2/\mu_R^2) \gamma^{(1)} - ( \beta_1 \ln(\mu_F^2/\mu_R^2) - \beta_0^2 \ln^2(\mu_F^2/\mu_R^2) )  \gamma^{(0)}
+    * it naturally incorporates renormalization group invariance,
+      as the perturbative order increases, estimates of MHOU decrease;
+    * the same procedure can be used for any perturbative process,
+      since the scale dependence of the strong coupling :math:`a_s(\mu^2)` and of PDFs is universal;
+
+However there is no unique principle to determine the specific range of the scale variation;
+the most used prescription specify to vary the factor :math:`\mu_F/\mu_R` in the range:
+:math:`1/2 \le \mu_F/\mu_R \le 2`.
+
+This variation can be performed at least at two different levels during the |PDF|
+evolution, always evaluating the strong coupling at :math:`\mu_R^2`.
+
+    * For ``SV_scheme='A'`` the variation is applied directly to the splitting functions
+      and the anomalous dimension are then modified using :cite:`Vogt:2004ns`:
+
+    .. math ::
+        & \gamma^{(1)}(N) \to \gamma^{(1)}(N) - \beta_0 L \gamma^{(0)} \\
+        & \gamma^{(2)}(N) \to \gamma^{(2)}(N) - 2 \beta_0 L \gamma^{(1)} - ( \beta_1 L - \beta_0^2 L^2) \gamma^{(0)} \\
+        & L= \ln(\mu_F^2/\mu_R^2)
+
+    * In ``SV_scheme='B'`` the |EKO| is multiplied by an additional kernel, such that
+      the scale variation is applied to the whole |PDF| set:
+
+    .. math ::
+        \tilde{\mathbf{E}}(a_s \leftarrow a_s^0) = \left[ 1 - L \gamma +L^2(\gamma^{2} + \frac{\partial \gamma}{\partial a_s} \beta(a_s)) + ... \right] \tilde{\mathbf{E}}(a_s \leftarrow a_s^0)
+
+    where scale variation kernel is expanded consistently order by order in :math:`a_s`,
+    leading to:
 
+    .. math ::
+        1 - L \gamma +L^2(\gamma^{2} + \frac{\partial \gamma}{\partial a_s} \beta(a_s)) =
+        1 - a_s L \gamma^{(0)} + a_s^2 \left [ L \gamma^{(1)} + \frac{1}{2} L^2 \gamma^{(0)} (\gamma^{(0)}+\beta_0) \right ]
 
-while keeping the evaluation of the strong coupling always at :math:`\mu_R^2`.
-Estimating the theoretical uncertainties imposed on |PDF| determination due to
-missing higher order corrections using scale variation in the evolution
-corresponds to schemes A and B in :cite:`AbdulKhalek:2019ihb`.
+    In this way the dependence of the |EKO| on :math:`L` is factorized outside the unvaried evolution kernel.
 
+Notice that the two methods should be equivalent, especially for fully linearized solutions
+(`ModEv=truncaded,ev_op_max_order=1`)
```

Comment:
`truncated` doesn't use `ev_op_mak_order`, does it? it is linear by construction ...

Reply:
Nope, `TRN` does use `ev_op_max_order`, and indeed we found that it was a major flaw in the comparison. Furthermore, at the time we found that the correct value was `ev_op_max_order=2`, and because of that we corrected a bug in the counting, now the correct value is actually `1`.

Reply:
Nope again ;-)
- the bug was on `ev_op_max_iterations`: we needed to do the whole evolution in a singlet step per patch
- instead: `ev_op_max_order` is used in the `perturbative-*` strategies of the singlet to determine the maximum coefficient of `U` https://eko.readthedocs.io/en/latest/theory/DGLAP.html#nlo-singlet-evolution https://github.com/N3PDF/eko/blob/d02dbe839f02d43915f54650fe3dbad83a583d0b/src/eko/kernels/singlet.py#L314

Reply:
Okay, I confused them as well, so I meant `ev_op_max_iterations`